app/controllers/lightweight_activities_controller.rb

```diff
@@ -103,8 +119,8 @@ def summary
     authorize! :read, @activity
     current_theme
     current_project
-    if !params[:response_key]
-      redirect_to summary_with_response_path(@activity, @session_key) and return
+    if !params[:run_key]
+      redirect_to summary_with_run_path(@activity, @run_key, request.query_parameters) and return
```

Comment:
I've confirmed that the Roadmap folks are not using this approach to generate run keys. I think this code was here to help when a run key was stored with the session or the user. Now that session based run keys are gone, this code makes less sense. And it might even be better to prevent the creation of an anonymous run_key when visiting the summary page.